bin/si/src/args.rs

```diff
@@ -50,6 +50,15 @@ pub(crate) struct Args {
     /// Show a preview of what the System Initiative Launcher will do
     #[arg(long, short = 'p', default_value = "false")]
     pub is_preview: bool,
+    
+    /// Allows starting the service and binding to a specific IP
+    #[arg(long = "host", default_value = "127.0.0.1")]
+    pub bind_host: String,
+
+    /// Allows starting the service and binding to a specific port
+    #[arg(long = "port", default_value = "8080")]
```

Comment:
Is this for the web port or the sdf port or both?

Reply:
web port only -- probably can make that more clear by specifying it precisely, but I wasn't sure what (if any) strategy there should be more holistically for overriding/customizing all the ports and binds.

Reply:
I think that would be a good distinction to make so that we can then expose the ability to do the same with SDF :) Maybe web_port would be a good name?